docker/docker-compose.yml

```diff
@@ -411,21 +391,14 @@ services:
       - ./volumes/db/pooler.sql:/docker-entrypoint-initdb.d/migrations/99-pooler.sql:Z
       # Use named volume to persist pgsodium decryption key between restarts
       - db-config:/etc/postgresql-custom
+    ports:
+      - '${POSTGRES_PORT}:${POSTGRES_PORT}'
```

Comment:
**question**

Since we have the supavisor container handling the pooling bellow, is it necessary to expose the postgres container port directly ?

Reply:
ah true, probably not. I would prefer to expose both with PG in the default port but I don't have a strong opinion. 
Happy to remove if we only want one?